hypershift-operator/controllers/nodepool/kubevirt.go

```diff
@@ -1,39 +1,168 @@
 package nodepool
 
 import (
+	"fmt"
+
 	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
 	corev1 "k8s.io/api/core/v1"
+	apiresource "k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kubevirtv1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
 	capikubevirt "sigs.k8s.io/cluster-api-provider-kubevirt/api/v1alpha1"
 )
 
+const (
+	defaultMemory       = "4Gi"
+	defaultCores        = uint32(2)
+	defaultVolumeSizeGi = 16
```

Comment:
have you considered using https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource to manage this input?

Reply:
The api is using a `resource` object. I just used an int here because it couldn't figure out how to make this default a const if it was a `resource` object. 

For example, this compiles.

```
var (
        defaultVolumeSize = apiresource.MustParse("16Gi")
)
```

this does not compile.

const (
        defaultVolumeSize = apiresource.MustParse("16Gi")
)